s29744-JakubMarkowski/zadania 1-14 30.09.txt

```diff
@@ -0,0 +1,86 @@
+
+Zadania wykonuje za pomocą VirtualBox.
+
+1.Uruchom terminal.
+
+
+1.prawy przycisk myszy, open in terminal.
+2.menu, terminal.
+3.sprawdziłem w internecie, jest możliwość ctrl + alt + T
+
+
+2.Wyświetl zawartość aktualnego katalogu.
+
+ls
+
+Wyświetla zawartość katalogu
+
+3.Sprawdź gdzie aktualnie się znajdujesz.
+
+pwd
+
+Wyświetla aktualną ścieżke
+
+4.Przejdź do katalogu Pulpit za pomocą ścieżki względnej
+
+cd Desktop/
+
+5.Sprawdź gdzie jesteś.
+
+pwd
+
+6.Wyświetl zawartość bieżącego katalogu.
+
+ls 
+
+7.Wyświetl zawartość swojego katalogu domowego bez przechodzenia do niego (ls z odpowiednim parametrem).
+
+ls ~
+
+przechodze do katalogu domowego
+
+8.Przejdź do katalogu Obrazy w twoim katalogu domowym za pomocą ścieżki względnej.
+
+cd ~/Music/
+
+cofam się do katalogu domowego i wchodze do Music
+
+9.Sprawdź gdzie jesteś.
+
+pwd
+
+10.Zobacz do jakiego katalogu przejdziesz za pomocą ścieżki ././././././
+W opisie rozwiązania proszę umieścić również to jak rozumiesz co się właściwie stało.
+Dlaczego znalazła/eś się w tym miejscu do którego przeniosła Cię ta ścieżka?
+Czy dodanie kilku ./././././ na początku/środku/końcu powoduje, że ścieżka się zmieni?
+Co ta ścieżka oznacza/sobą reprezentuje?
+
+-u mnie po wpisaniu cd ././././././ nic się nie stało, ponieważ ./ jest moim obecnym katalogiem. Zauważyłem, że polecenie cd ././././home
```

Comment:
No nie "nic się nie stało" tylko ścieżka wynikowa nie różni się od ścieżki źródłowej ale się stało i to dużo.
Jednakże zapis "cd ././././home
przeniesie mnie do home."
mnie dziwi bo to może zadziałać tylko w jednym przypadku jeśli Twoim katalogiem bieżącym jest `/`.
Z poprzednich poleceń wynika, że powinieneś być w katalogu `/home/LOGIN/Music` w którym standardowo nie znajduje się kolejny katalog `home`.

Martwi mnie to, że "zauważyłeś", że to polecenie Cię przeniesie do home. To nie ma prawa zadziałać.
Chyba, że zapiszesz to tak `/././././home` (ze slashem na początku) to wtedy tak.
Ale tu wtedy wychodzi nieuwaga.
Proszę skoryguj swoje rozwiązanie.